src/js/lib/logging/server.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express')
+const fs = require('fs')
+const uuid = require('uuid')
+const yargs = require('yargs')
+
+
+module.exports = {
+    startServer
+  }
+
+
+function main(argv) {
+    startServer(parse_args(argv).port)
+}
+
+
+function parse_args(argv) {
+    return yargs(argv)
+        .option('port', {
+            alias: 'p',
+            description:
+                'The number of the port that this server will listen on. ' +
+                'If the the number is 0 then an arbitrary free port will be chosen.',
+            type: 'number',
+            default: 20060
```

Comment:
This literal is used in 4 different places across different files. Please use literals only once, always.